src/db/auth.ts

```diff
@@ -87,7 +87,7 @@ export async function createUser(
 	try {
 		const [res] = await pool.execute<ResultSetHeader>(
 			`INSERT INTO user (first_name, last_name, email, password_hash, is_admin)
-            VALUES (:first_name, :last_name, :email, :hash, 1)`,
+            VALUES (:first_name, :last_name, :email, :hash, 0)`,
 			{ first_name, last_name, email, hash },
 		);
```

Comment:
Let's make this field `DEFAULT false` and omit this from the function entirely.

Reply:
i used TINYINT as the type will it cause an error?

Reply:
I mean it shouldn't because booleans are implemented as `TINYINT(1)` under the hood, but lets just make the column a `BOOLEAN` type for clarity.